apps/src/signUpFlow/FinishTeacherAccount.tsx

```diff
@@ -10,10 +10,15 @@ import {
 } from '@cdo/apps/componentLibrary/typography';
 import SchoolDataInputs from '@cdo/apps/templates/SchoolDataInputs';
 
+import {navigateToHref} from '../utils';
+
 import locale from './locale';
 import {
-  DISPLAY_NAME_SESSION_KEY,
-  EMAIL_OPT_IN_SESSION_KEY,
```

Comment:
Realized it probably doesn't make sense to use `sessionStorage` for the variables set on the same page it's being submitted so removing some of those here! (will remove the student-specific ones in the followup PR that creates the student user)